sklearn/decomposition/_incremental_pca.py

```diff
@@ -396,11 +396,12 @@ def transform(self, X):
         """
         if sparse.issparse(X):
             n_samples = X.shape[0]
-            output = []
-            for batch in gen_batches(
-                n_samples, self.batch_size_, min_batch_size=self.n_components or 0
-            ):
-                output.append(super().transform(X[batch].toarray()))
+            output = [
+                super(IncrementalPCA, self).transform(X[batch].toarray())
```

Comment:
`super()` doesn't need args

Reply:
Ah, my wrong. Rather it appeared not to work in this case, and I added them in the hope errors would go away.

Reply:
Fixed.